configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,17 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner contains a local vulnerability definitions database. If {rh-rhacs-first} is running in normal mode, Scanner fetches new vulnerability definitions from Central. Central retrieves the updates from the internet, and then updates the Central database.
 //TODO: Add link to Vulnerability definitions
 
-However, when you are using {product-title} in offline mode, you must manually update Scanner definitions by uploading them to Central.
+However, if you are using {product-title-short} in offline mode, you must manually update the definitions. 
+By uploading the definitions to Central, you can manually update the definitions.
 
-When {product-title} runs in offline mode, Scanner checks for new definitions from Central. If new definitions are available, Scanner downloads the new definitions from Central, marks them as default, and then uses the updated definitions for scanning images.
+If the product is running in offline mode, Scanner checks with Central for new data approximately every 5 minutes. Central also checks for new data approximately every 5 minutes. 
```

Comment:
In both online and offline mode, Scanner checks with Central every 5 minutes, by default.

In online mode, Central checks for new data every 5 - 20 minutes (can probably get away with just saying approximately 5).